src/notation/internal/notationuiactions.cpp

```diff
@@ -213,7 +214,7 @@ const UiActionList NotationUiActions::m_actions = {
     UiAction("toggle-visible",
              mu::context::UiCtxNotationOpened,
              QT_TRANSLATE_NOOP("action", "Toggle visibility of elements"),
-             QT_TRANSLATE_NOOP("action", "Toggle visibility of elements")
+             QT_TRANSLATE_NOOP("action", "Toggle visibility of elements"),
```

Comment:
You've an extra comma here, that's why the test builds are failing! Please amend your commit and force-push, then the checks will be run again and everything will be all right.

For the rest, it looks great!

Reply:
Sure this comma is the culprit?

Reply:
Yes I am. Strange discrepancy in C++: in an enum or list, a comma before a `}` is allowed, but before a `)` it gives an error.

Reply:
OK, so that's 2 build problems then, the other one needs to get fixed too